docs/debian-conf.d/transport/30_vexim_virtual_delivery

```diff
@@ -1,6 +1,6 @@
 
 ### transport/30_vexim_virtual_delivery
-#################################
+#######################################
 
 virtual_delivery:
```

Comment:
In fact, the transports should probably be split into multiple files. It seems our file is the only one which defines multiple transports. As for the naming, OK, you can leave it as is.